server/handler/options.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022 Palantir Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handler
+
+import (
+	"github.com/palantir/bulldozer/bulldozer"
+	"os"
+	"strconv"
+)
+
+const (
+	DefaultSharedRepository = ".github"
+	// DefaultConfigurationPath
+	// The default configuration path is the same for the repo as
+	// for the shared config if not configured
+	DefaultConfigurationPath = "bulldozer.yml"
+	DefaultAppName           = "bulldozer"
+)
+
+type Options struct {
```

Comment:
Any particular reason to move this type to the `handler` package or was this to mirror how we did it in `policy-bot`?

Reply:
I implemented the way, to mirror how you did it `policy-bot`. Because then it's a bit more separated from the main config. If you want I can move it all the main file.